src/Sulu/Bundle/PreviewBundle/Resources/js/views/Preview/Preview.js

```diff
@@ -0,0 +1,265 @@
+// @flow
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {action, autorun, observable, toJS} from 'mobx';
+import {observer} from 'mobx-react';
+import debounce from 'debounce';
+import Router from 'sulu-admin-bundle/services/Router';
+import {sidebarStore} from 'sulu-admin-bundle/containers/Sidebar';
+import {FormStore} from 'sulu-admin-bundle/containers/Form';
+import ResourceStore from 'sulu-admin-bundle/stores/ResourceStore';
+import Requester from 'sulu-admin-bundle/services/Requester';
+import Toolbar from 'sulu-admin-bundle/components/Toolbar';
+import Loader from 'sulu-admin-bundle/components/Loader';
```

Comment:
Usually we only import using only one folder deep, i.e. `sulu-admin-bundle/components` or `sulu-admin-bundle/containers`, since we consider this the "public" interface of our bundles.